Makefile

```diff
@@ -33,48 +34,33 @@ clean: ## cleans up the repository
 	/bin/rm -rf $(PKGDIR)
 	/bin/rm -rf $(DATADIR)
 
-build: $(addprefix build/,$(DISTRIBUTIONS)) ## builds the code for all distributions
-
-test: $(addprefix test/,$(DISTRIBUTIONS)) ## runs tests for all distributions
-
-test/%: TEST_FLAGS ?= -v
-test/%: vet ## runs unit tests for a specific distribution
-	@echo "*\n* Testing for $* locally \n*"
-	go test $(TEST_FLAGS) -tags $*
+test: vet 
+	go test -v ./...
 
 format: ## formats the code
 	gofmt -w $(SOURCE)
 
 vet: ## examines the go code with `go vet`
-	go vet $(SOURCE)
+	go vet ./...
 
 up: $(addprefix up/,$(DISTRIBUTIONS)) ## start agents for all distributions
 up/%: ## starts the agent for a specific distribution
```

Comment:
Can you also do a slight change here? If you change `up/%:` to `up/%: build` we can also stop building custom images for all the repos, just mount `build` folder to them in docker-compose and execute the binary in them.
In short, can you change this so that we run the generated binary inside docker containers and not run the source code with go installed.